src/nuget.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
+    <add key="platform.uno dev" value="https://uno-platform.pkgs.visualstudio.com/Uno%20Platform/_packaging/unoplatformdev/nuget/v3/index.json" />
```

Comment:
intended?

Reply:
Yes. Ultimately, all dev packages will be provided through the uno dev feed.